crates/blockless-drivers/Cargo.toml

```diff
@@ -33,6 +33,7 @@ rust-s3 = "0.32.0"
 futures-core = "0.3.25"
 futures-util = "0.3.25"
 md5 = "0.7.0"
+serde_json = "1.0.105"
```

Comment:
surprised to hear this - since this is a widely used crate: https://crates.io/crates/serde_json

I could not find anything on this; how to best test this on platforms other than Mac (which im using)?